db/migrate/20210301230035_create_users.rb

```diff
@@ -1,10 +1,13 @@
+# frozen_string_literal: true
+
 class CreateUsers < ActiveRecord::Migration[6.1]
   def change
     create_table :users do |t|
       t.string :first_name
       t.string :last_name
       t.string :email
-      
+      t.string :role
```

Comment:
Why did you change this file? You should only add to existing migrations (like you did below) but never change existing ones.